src/components-examples/material/sidenav/sidenav-responsive/sidenav-responsive-example.html

```diff
@@ -1,25 +1,21 @@
 @if (shouldRun) {
-  <div class="example-container" [class.example-is-mobile]="mobileQuery.matches">
+  <div class="example-container" [class.example-is-mobile]="mobileQuery">
     <mat-toolbar color="primary" class="example-toolbar">
-      <button mat-icon-button (click)="snav.toggle()"><mat-icon>menu</mat-icon></button>
+      <button mat-icon-button (click)="snav.toggle()">
+        <mat-icon>menu</mat-icon>
+      </button>
       <h1 class="example-app-name">Responsive App</h1>
     </mat-toolbar>
 
-    <mat-sidenav-container class="example-sidenav-container"
-                          [style.marginTop.px]="mobileQuery.matches ? 56 : 0">
-      <mat-sidenav #snav [mode]="mobileQuery.matches ? 'over' : 'side'"
-                  [fixedInViewport]="mobileQuery.matches" fixedTopGap="56">
+    <mat-sidenav-container class="example-sidenav-container" [style.marginTop.px]="mobileQuery ? 56 : 0">
+      <mat-sidenav #snav [mode]="mobileQuery ? 'over' : 'side'" [fixedInViewport]="mobileQuery" fixedTopGap="56">
         <mat-nav-list>
-          @for (nav of fillerNav; track nav) {
-            <a mat-list-item routerLink=".">{{nav}}</a>
-          }
+          <a mat-list-item *ngFor="let nav of fillerNav">{{ nav }}</a>
```

Comment:
Why was this switched to `ngFor`? The whole repo intentionally uses the new control flow syntax.